test/data/README.md

```diff
@@ -46,6 +46,21 @@ Redistributing the dataset "taxi-fare-test.csv", "taxi-fare-train.csv" with attr
 > 
 > The dataset is provided under terms provided by City of New York: https://opendata.cityofnewyork.us/overview/#termsofuse.
 
+### MSLR-WEB10K, MSLR-WEB30K
+
+This dataset is originally from [Introducing LETOR 4.0 Datasets](https://arxiv.org/abs/1306.2597).
+The dataset is under a CC-by 4.0 license.
+> @article{DBLP:journals/corr/QinL13,
+>  author    = {Tao Qin and Tie{-}Yan Liu},
+>  title     = {Introducing {LETOR} 4.0 Datasets},
+>  journal   = {CoRR},
+>  volume    = {abs/1306.2597},
+>  year      = {2013},
+>  url       = {https://arxiv.org/abs/1306.2597},
+>  timestamp = {Mon, 01 Jul 2013 20:31:25 +0200},
+>  biburl    = {http://dblp.uni-trier.de/rec/bib/journals/corr/QinL13},
```

Comment:
(minor)
Recommend switching to HTTPS links. I see two HTTP links in the citation which can use a secure link.